kubectl-minio/cmd/version.go

```diff
@@ -64,6 +75,40 @@ func newVersionCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 
 // run initializes local config and installs MinIO Operator to Kubernetes cluster.
 func (o *operatorVersionCmd) run() error {
-	fmt.Println(version)
+	fmt.Println("Kubectl-Plugin Version:", version)
+	cfg := config.GetConfigOrDie()
```

Comment:
is really necessary to exit if client creation fails? probably we should out only the plugin binary version in case of no connection to k8s

Reply:
nevermind, read it again and it;s alrady there